queries/agenda.js

```diff
@@ -31,6 +31,7 @@ const fetchFilesFromAgenda = async (agendaId, currentUser) => {
       ?document besluitvorming:vertrouwelijkheidsniveau ?confidentialityLevel .`
   }
   queryString += `
+    VALUES ?extension { ${extensions.map(extension => sparqlEscapeString(extension)).join(" ")} }
```

Comment:
`extensions.map` doesn't work if `extensions` is `[]` (which is possible from app.js and from api call with data)
So wrap with an if statement.

Reply:
Unsure how the frontend component is going to work in a scenario with multiple types.
If the incoming API call has no extensions, we default to pdf. (filter on VALUES)
If the incoming API call has extensions but the list is empty, I would argue that means all types. (no filter on VALUES)
If the incoming API call has extensions and not empty, I would argue that means specified types. (filter on VALUES)
The last one has a risk of typo's. "PDF" vs "pdf" should yield different query results? not sure if resource checks capitals

Reply:
Would scope this to:
- add check for `extensions  === []` around this statement
- make `["pdf"] default
- add `toLowerCase()` to all extensions to minimize risk of capitalization

Out of scope:
- make sure that when a document is uploaded through Kaleidos, the extension is lowercased (might already be done this way, but have to check)